src/_includes/docs/swift-package-manager/migrate-ios-project-manually.md

```diff
@@ -0,0 +1,84 @@
+Once you [turn on Swift Package Manager][] support, the Flutter CLI tries to
+migrate your project to add Swift Package Manager integration.
+This makes your project download the Swift packages that your Flutter
```

Comment:
maybe a bit more clear to say "the Flutter tooling downloads the Swift packages" ?

Reply:
Swift packages are downloaded by Xcode. The Flutter tool just generates some glue.

The process is roughly:

1. The Flutter tool download pub packages
2. The Flutter tool generates code to add iOS/macOS plugins to the Xcode project
3. Xcode downloads the Swift package dependencies for each iOS/macOS plugin

Before SwiftPM, the generated code in step 2 was CocoaPods magic. Now, the generated code also includes SwiftPM magic. The "Swift Package Manager integration" is your project including the SwiftPM magic that was generated by step 2.